components/pages/file-entity/useEntityData.tsx

```diff
@@ -45,26 +49,28 @@ const useEntityData = ({ fileId }: { fileId: string }): EntityData => {
   });
 
   if (loading) {
-    return { programShortName: null, data: null, loading: true };
+    return { ...noData, loading: true };
   } else {
     const entity = get(data, 'file.hits.edges[0].node', null);
 
     if (!entity) {
-      return { programShortName: null, data: null, loading: false };
+      return { ...noData, loading: false };
     }
 
     const programShortName = entity.study_id;
+    const size = get(entity, 'file.size', 0);
+    const access = entity.file_access;
 
     const summary: FileSummaryInfo = {
       fileId: entity.file_id,
       objectId: entity.object_id,
       fileFormat: entity.file_type,
-      size: get(entity, 'file.size'),
-      access: entity.file_access === 'controlled' && FileAccessState.CONTROLLED,
+      size: size,
+      access: entity.file_access,
```

Comment:
minor thing, but you already made constants for these two:

```suggestion
      size,
      access,
```